hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -247,6 +242,65 @@ NETCONSOLE=6665@169.254.1.1/eth0,6666@/
 
 Version: pieeprom-2020-06-15.bin - STABLE  
 
+### USB_MSD_EXCLUDE_VID_PID
+A list of up to 4 VID/PID pairs specifying devices which the bootloader should ignore. If this matches a HUB then the HUB won’t be enumerated, causing all downstream devices to be excluded.
+This is intended to allow problematic (e.g. very slow to enumerate) devices to be ignored during boot enumeration. This is specific to the bootloader and is not passed to the OS.
+
+The format is a comma-separated list of hexadecimal values with the VID as most significant nibble. Spaces are not allowed.
+E.g. `034700a0,a4231234`
```

Comment:
Perhaps this should be expanded slightly, to something like:
E.g. `USB_MSD_EXCLUDE_VID_PID=034700a0,a4231234` will exclude a USB device with VID 0x0347 & PID 0x00a0 and a USB device with VID 0xa423 and PID 0x1234